README.md

```diff
@@ -1,12 +1,45 @@
 # discord.py-welcome-bot
 
-A simple bot that can be used to welcome new users to a discord server, sending them a custom private message and adding them to a role.
+A simple bot that can be used to welcome new users to a discord server, send them a custom private message, send a welcome message to a specific channel, and adding them to a role.
 
 A video explaining how to set up everything except adding the user to a role can be found [here](https://youtu.be/N0NP7BfUFxA).
 
-**Update November 2019** - Now set up as a docker container to ensure full compatibility as I will not be maintaining this. Specific versions of Python and Discord.py are required for this script to work.
-
-## How to install the welcome-bot as a docker container on a server (raspberrypi)
+**Update November 2019** - Now set up as a docker container to ensure full compatibility as I will not be maintaining this. Specific versions of Python and Discord.py are required for this script to work.\
+**Update November 2020** - Added basic install instructions for home computers and generic Linux distros, and how to enable Intents.
+## Intents
```

Comment:
Markdown headings should have a blank line above and below them. I'm conscious I didn't do this for this project, and I am deeply ashamed.